### PR TITLE
fix: do not error on existing `/tmp/src` during prep

### PR DESCRIPTION
**Pull request by gonzolively**
Prep script kept exiting due to the existence of "/tmp/src" which I found very annoying, so added a check for it and subsequent removal of it if found. This may not be desired, but might be worth merging.

### CODE REVIEW

File: scripts/prep.sh [hunk 1]
```diff
@@ -24,4 +24,9 @@ docker pull rancher/k3d-tools:5.0.0
 docker pull python:3.7-slim
 
 section "Preparing Filesystem..."
+
+if [ -d "/tmp/src" ]; then
+  rm -rf /tmp/src
+fi
+
 mkdir /tmp/src
```
Comment (iwilltry42) [inline scripts/prep.sh]: Just make `mkdir` not yield an error instead :)

```suggestion
if [ -d "/tmp/src" ]; then
  rm -rf /tmp/src
fi

mkdir -p /tmp/src
```

**Review by iwilltry42 — CHANGES_REQUESTED** [review]

**Comment by iwilltry42** [comment]
Thanks for the heads-up @gonzolively :pray:

**Comment by gonzolively** [comment]
Of course, thanks for reviewing/merging @iwilltry42!

Also, this is probably not the best place for this conversation, but I wasn't able to find any exposed services/URLs from the demo app. I'm unsure how to use the demo, besides spinning up a cluster and destroying it with k3d. This may be an education problem on my side though.

**Comment by iwilltry42** [comment]
The `lifecycle` script should call chromium to open the sample app, see https://github.com/k3d-io/k3d-demo/blob/main/scripts/lifecycle.sh#L41

You may need a `hosts` file entry so that `.localhost` addresses resolve to `127.0.0.1` on your host machine. On MacOs/Linux this can automatically be done with the `libnss-myhostname` package.

**Comment by gonzolively** [comment]
Thanks, I now have the demo up and running! 

I assumed (wrongly or rightly) that I could access the demo using `demo-multiserver.`

**Pull request merged**